corehq/couchapps/apps_with_submissions/views/view/map.js

```diff
@@ -0,0 +1,7 @@
+function(doc){
+    if((doc.doc_type == 'Application' || doc.doc_type == 'RemoteApp') &&
+            doc.copy_of != null &&
```

Comment:
we could potentially have apps that are not copies that have submissions (i think). this would happen with preview forms. i didn't worry about this case because it doesn't affect exports as we always analyze the current app